e2e/specs/add-appointments.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { expect } from '@playwright/test';
+import { generateRandomPatient, deletePatient, type Patient } from '../commands';
+import { test } from '../core';
+import { AppointmentsPage } from '../pages';
+
+let patient: Patient;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+});
+
+test('Add appointment for a patient, edit the added appointment and cancel it', async ({ page, api }) => {
+  const appointmentsPage = new AppointmentsPage(page);
+  await test.step('When I click on appointment tab', async () => {
+    await appointmentsPage.goto(patient.uuid);
+  });
+  //test to add appointment
+  await test.step('And I click on the “Add” button', async () => {
+    await page.click('button:has-text("Add")');
+  });
+
+  await test.step('When I select Mobile Clinic location', async () => {
+    await page.locator('#location').selectOption('8d9045ad-50f0-45b8-93c8-3ed4bce19dbf');
```

Comment:
Please refer to @ibacher 's comment on slack:

> Always prefer grabbing the element by user-facing text (in this case the ARIA label) rather than technical details like id / class / tag as we want our tests to be robust to markup changes.

Reply:
@jayasanka-sack There's no element under options other than changing their values of location. 
We have used the same [here](https://github.com/Vijaykv5/openmrs-esm-patient-chart/blob/main/e2e/specs/program-enrollment.spec.ts#L26)

Reply:
Can we try with something akin to what's showcased in the Playwright documentation ([link](https://playwright.dev/docs/input#select-options))?

`await page.getByLabel('Choose a color').selectOption({ label: 'Blue' });`

Reply:
This applies for other fields as well.

Reply:
Oops I missed that!
Fixed now :)
Thanks